packages/drivers/odsp-driver/src/contracts.ts

```diff
@@ -225,3 +226,8 @@ export interface IFlushOpsResponse {
     retryAfter?: number;
     lastPersistedSequenceNumber?: number;
 }
+
+export interface ISnapshotResponse {
```

Comment:
I think you can avoid bumping version for cache entries by having 
```typescript
export interface ISnapshotResponse extends ISnapshotContents {
   cacheEntryTime?: number;
}
```
Also I assume we want it to be required filled in the future, right?

Reply:
Also name is a bit not on point - "response" is more applicable to fetch calls, here it should maybe be called ISnapshotCachedEntry? Or something like that?

Reply:
Changed the name. Also If I upgrade the version, I can make sure that the time exists. Otherwise currently for older entries it could be undefined and cannot assert as I am asserting currently during logging. 
But I think invalidating cache entries can cause a little performance blip, so I can remove the assert.